capa/features/address.py

```diff
@@ -88,29 +89,54 @@ def __eq__(self, other):
         return (self.process, self.tid) == (other.process, other.tid)
 
     def __lt__(self, other):
+        assert isinstance(other, ThreadAddress)
         return (self.process, self.tid) < (other.process, other.tid)
 
 
-class DynamicAddress(Address):
+class CallAddress(Address):
```

Comment:
i wonder if we should call this `DynamicCallAddress` like we use `DynamicReturnAddress` below?

Reply:
Hmm, I agree. I had the general future case in mind where the call scope would be also part of static analysis. But given that this address has the thread attribute right now, maybe we should use `DynamicCallAddress`. Perhaps we can introduce a `StaticCallAddress` when the call scope is added to static analysis.